### PR TITLE
Move AzureConnection's strings into localizable resources

### PR DESCRIPTION
**Pull request by DHowett-MSFT**
## Summary of the Pull Request
This also sets up TerminalConnection to _have_ resources, which will be useful for the messaged in #3461.

**This pull request does not adjust any of the state machine logic, but a following one will.**

## References
This was untracked work that was mentioned in passing in #2146 .

## PR Checklist
* [ ] Closes <nothing>
* [x] CLA signed
* [x] Tests added/passed/manually tested
* [ ] Requires documentation to be updated
* [x] Core

## Validation Steps Performed
Walked through the Azure connection myself.

### CODE REVIEW

**Comment by DHowett-MSFT** [comment]
Please don't comment on the ResX schema comment. It's not removable. VS just adds it back every single time you save.

**Comment by DHowett-MSFT** [comment]
Yes, even with its spelling errors.

File: src/cascadia/TerminalConnection/AzureConnection.cpp [hunk 1]
```diff
@@ -4,9 +4,9 @@
 #include "pch.h"
 #include "AzureConnection.h"
 #include "AzureClientID.h"
-#include "AzureConnectionStrings.h"
 #include <sstream>
```
[the remark this answers is not in the document]
Comment (miniksa) [reply src/cascadia/TerminalConnection/AzureConnection.cpp]: (Or we can fork it to another PR... as I don't have any other comments here.)

[the remark this answers is not in the document]
Comment (DHowett-MSFT) [reply src/cascadia/TerminalConnection/AzureConnection.cpp]: I'll do a followup. sstream is definitely in LibraryIncludes, and cstdlib/stdlib.h is not.

**Pull request merged**